modern/src/theme/dimensions.js

```diff
@@ -2,5 +2,7 @@ export default {
   inputHeight: '42px',
   borderRadius: '4px',
   sidebarWidth: '28%',
-  tabletSidebarWidth: '52px' 
+  tabletSidebarWidth: '52px',
+  drawerWidth: 360,
+  tabletDrawerWidth: 320,
```

Comment:
Should these be pixels? Does `drawerWidth` refer to desktop version? Should we make it explicit?

Reply:
the desktop version is 360. Mobile and tablet is 320